openedx/features/course_experience/templates/course_experience/course-home-fragment.html

```diff
@@ -64,32 +64,37 @@ <h2 class="hd hd-3 page-title">${course.display_name_with_default}</h2>
                     </div>
                 % endif
 
-                ${HTML(outline_fragment.body_html())}
+                % if outline_fragment:
+                    ${HTML(outline_fragment.body_html())}
+                % endif
             </main>
             <aside class="course-sidebar layout-col layout-col-a">
                 <%
-                course_tools = CourseToolsPluginManager.get_course_tools()
+                course_tools = filter(
```

Comment:
We don't have a policy around this. For me, it seemed simpler to co-locate the logic, but I can see benefits to simplifying the template. Note that Django Templates don't support arbitrary code evaluation so then the tools would have to be passed in.

I'm going to leave this as is, unless you disagree.